src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/LoongArch64/LA64Base.PlatformNotSupported.cs

```diff
@@ -0,0 +1,217 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable IDE0060 // unused parameters
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.LoongArch64
```

Comment:
I've marked this as `NO-MERGE` since we cannot take it until after an API review has occurred. See https://github.com/dotnet/runtime/blob/main/docs/project/api-review-process.md

We need an API proposal, following the standard template, created first. We'll have the discussion on relevant name changes and other bits there, then I can then champion that and take it to API review. Once approved, we can then implement the API surface.

Until then, LoongArch would be relegated to only supporting the existing cross platform API surface. For example, `Leading/TrailingZeroCount` can be supported by accelerating `int.Leading/TrailingZeroCount` and the same methods on the other primitive types.

Reply:
Thanks !
Reviewing the API for LoongArch64 based on a PR maybe more clear. So I pushed this PR.
I will create an API proposal for LoongArch64's API.